infra/brilift.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
```

Comment:
Is this copied from somewhere? We should at least link to where it came from.
Perhaps the crate could expose it somehow, as a string or file? It may get out of sync at some point

Reply:
It's copied from [https://github.com/sampsyo/bril/blob/main/brilift/rt.c]. I don't know how to keep it synced though.

Reply:
Could brillift have a function that spits out the string using the `include_str` macro?

Reply:
https://github.com/sampsyo/bril/pull/314